src/db/index.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+const fs = require('fs');
+const util = require('util');
+const path = require('path');
+
+const readFile = util.promisify(fs.readFile);
+// const writeFile = util.promisify(fs.writeFile);
+
+const productsPath = path.resolve('db/products.json');
+
+const register = (newUser) => {
+  fs.readFile('./src/db/users.json', 'utf8', (err, data) => {
+    if (err) throw err;
+    else {
+      const users = JSON.parse(data);
```

Comment:
It is good you used **const** here for users rather than **let** as that does not change in the function. Good job!